src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -37,6 +37,7 @@ object Backend {
  * Trait to be implemented by concrete backends.
  */
 trait Backend {
+  type A <: BackendCall
```

Comment:
Nice!  Though maybe a more descriptive name for the type variable like `BC`?  I know some people who would advocate for `BACKEND_CALL` but I'm not going that far.  :stuck_out_tongue: